.github/workflows/dev-embedded-mcl-ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: gradle build with packaged mcl
```

Comment:
The names could be difficult for the badges in the readme. @JanBobolz could we shorten these?

Reply:
fixed by 4095694